plugins/leemons-plugin-calendar/frontend/src/helpers/getCalendarNameWithConfigAndSession.js

```diff
@@ -1,6 +1,8 @@
 export default function getCalendarNameWithConfigAndSession(calendar, config, session) {
   let label = calendar.name;
   if (config && config.userCalendar && config.userCalendar.id === calendar.id)
-    label = session?.name + (session?.surnames || '');
+    label = `${session?.surnames !== null ? session?.surnames : ''}${
+      session.surnames ? ', ' : ''
+    } ${session?.name}`;
```

Comment:
Remove double space.

```suggestion
    label = `${session?.surnames !== null ? session?.surnames : ''}${
      session.surnames ? ', ' : ''
    }${session?.name}`;
```

Reply:
removed! thx!